generators/client/templates/angular/src/main/webapp/app/polyfills.ts.ejs

```diff
@@ -1,42 +1,3 @@
-<%#
- Copyright 2013-2019 the original author or authors from the JHipster project.
-
- This file is part of the JHipster project, see https://www.jhipster.tech/
- for more information.
-
- Licensed under the Apache License, Version 2.0 (the "License");
- you may not use this file except in compliance with the License.
- You may obtain a copy of the License at
-
-      http://www.apache.org/licenses/LICENSE-2.0
-
- Unless required by applicable law or agreed to in writing, software
- distributed under the License is distributed on an "AS IS" BASIS,
- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- See the License for the specific language governing permissions and
- limitations under the License.
--%>
-/**
- * This file includes polyfills needed by Angular and is loaded before the app.
- * You can add your own extra polyfills to this file.
- *
- * This file is divided into 2 sections:
- *   1. Browser polyfills. These are applied before loading ZoneJS and are sorted by browsers.
- *   2. Application imports. Files imported after ZoneJS that should be loaded before your main
- *      file.
- *
- * The current setup is for so-called "evergreen" browsers; the last versions of browsers that
- * automatically update themselves. This includes Safari >= 10, Chrome >= 55 (including Opera),
- * Edge >= 13 on the desktop, and iOS 10 and Chrome on mobile.
- *
- * Learn more in https://angular.io/docs/ts/latest/guide/browser-support.html
- */
-
-/***************************************************************************************************
- * BROWSER POLYFILLS
- */
-
-/** IE9, IE10 and IE11 requires all of the following polyfills. **/
 import 'core-js/es6/symbol';
```

Comment:
recent release of `angular-cli` has removed all these pollyfills as these are now supported out of box by evergeeen browsers. Do we really require all these?

Reply:
That's the point of the PR, we load them only if necessary, so the evergreen browsers won't fetch this bundle, angular CLI does the same

Reply:
I thought we only support evergreen browsers and un-necessarily increases bundle size. This would also confuse developers on where to add new pollyfills that are also required in evergreen browsers.

As per my understanding, CLI doesn't add any support out of box and you need to manually add appropriate pollyfills to support old browsers. Do you have any code reference that points to this support by CLI?

Reply:
I don't know what we support officially, so I just found a solution to keep supporting older browsers without impacting evergreen browsers... Maybe we should discuss about the browser support, it can be better to simply remove this...

Here is the link for the angular CLI: https://blog.ninja-squad.com/2019/01/31/angular-cli-7.3/

Reply:
thanks for reference. Few points:
- I could see discussion on https://github.com/jhipster/generator-jhipster/issues/8250 related to browser support. May be v6 release is a good time to break officially.
- If we want to support, then, would prefer the angular cli approach to have specific polyfills containing only these and let main polyfills be containing others

Reply:
I guess it's better to break support, I prefer keeping generated code simple, if people want to add IE support, they can easily find out how to do it, we can even add a tip on the website to do this